.ci.yaml

```diff
@@ -382,8 +383,9 @@ targets:
       - flutter_frontend_server/**
 
   - name: Linux clangd
-    recipe: engine_v2/engine_v2
+    recipe: engine_v2/builder
     properties:
+      cores: "32"
```

Comment:
Copied from
https://github.com/flutter/engine/blob/38e37ce9879b2efe4890a8d09f4207dc5686a1bd/ci/builders/linux_unopt_debug_no_rbe.json#L12

Reply:
No, `drone_dimensions` in this case are properties that get passed to luci to decide which bots are eligible to run builds.  If you open the "Infra" tab of a build you can see the input properties.  For [example](https://ci.chromium.org/ui/p/flutter/builders/try/Linux%20clangd/6292/infra)
![Screenshot 2024-09-13 at 4 28 24 PM](https://github.com/user-attachments/assets/d8be62fb-97a5-4427-b74e-5c4619e0caeb)


`device_type=none` means no physically tethered device (devicelab), `os=Linux` is self-explanatory, and `cores=32` means the bot it runs on should have 32 cores.  

Come to think of it though, I actually don't think the `core=32` constraint is necessary for the clangd builder.  It was introduced in https://github.com/flutter/engine/pull/50901 and my guess is that it was copy/pasted from clang_tidy https://github.com/flutter/engine/pull/43448.

Since this test runs really fast even on the default 8 core machine, I think we probably shouldn't use the more "valuable" 32-core bots (40 bots in the pool) vs the 8-core machines (240 in the pool).  @matanlurey since this was a builder you created, wdyt?

Reply:
I agree, I definitely blindly copied this without thinking.